spec/features/employer/employer_can_login_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
@roscalabrin Yup, makes sense. Tests surrounding security and privacy are really important. It's common that if you are going to choose to *not* test something that you will choose to cut out sad paths. However, this is a sad path that could have severe consequences. It's also highly likely that someone will try to exploit it so testing it is really important.